pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -97,7 +103,9 @@ func NewNodeRouteController(
 		nodeLister:       nodeInformer.Lister(),
 		nodeListerSynced: nodeInformer.Informer().HasSynced,
 		queue:            workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(minRetryDelay, maxRetryDelay), "noderoute"),
-		installedNodes:   cache.NewIndexer(nodeRouteInfoKeyFunc, cache.Indexers{nodeRouteInfoPodCIDRIndexName: nodeRouteInfoPodCIDRIndexFunc})}
+		installedNodes:   cache.NewIndexer(nodeRouteInfoKeyFunc, cache.Indexers{nodeRouteInfoPodCIDRIndexName: nodeRouteInfoPodCIDRIndexFunc}),
+		PodCIDRIPsMap:    make(map[string]int),
```

Comment:
Looking through nodeRouteController code again. installedNodes (cache.indexer) already have this mapping functionality of searching using podCIDR through nodeRouteInfoPodCIDRIndexFunc. We do not need a separate map.

Reply:
Thanks, I updated code using nodeRouteInfoPodCIDRIndexFunc.